idea-plugin/META-INF/META-INF/plugin.xml

```diff
@@ -0,0 +1,35 @@
+<idea-plugin version="2">
+  <id>com.your.company.unique.plugin.id</id>
```

Comment:
This file shouldn't exist.  I'm guessing that IntelliJ generated it automatically for some reason. I'll remove.